config.js

```diff
@@ -0,0 +1 @@
+export const API_GATEWAY_LINK = 'https://1jeopqs6y0.execute-api.us-east-1.amazonaws.com/Dev/dummy';
```

Comment:
This doesn't solve the problem.  We still have the production server URL in the repo. We need a solution that allows this to be configured differently for `prod` and `dev`.